api/turing/api/appcontext.go

```diff
@@ -43,14 +43,15 @@ type AppContext struct {
 func NewAppContext(
 	db *gorm.DB,
 	cfg *config.Config,
+	apiPathPrefix string,
 	authEnforcer *enforcer.Enforcer,
```

Comment:
Can we instead of passing `apiPathPrefix` here, replace `authEnforcer` with `authorizer`? I.e. authorizer should initialized (if configured) outside of the app context. It doesn't make sense to initialize `authEnforcer` outside, but `authorizer` – inside of the app context, as they are used together.